Re : debrid-link.fr

SKEIDs opened the pull request:
first pull request : https://github.com/AdguardTeam/AdguardFilters/pull/102325

issues : https://github.com/AdguardTeam/AdguardFilters/issues/102235

SKEIDs committed:
debrid-link.fr

https://github.com/AdguardTeam/AdguardFilters/pull/102325

Sergey-Lyapin commented on AnnoyancesFilter/sections/cookies_specific.txt:
```diff
@@ -112,6 +112,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+debrid-link.fr###cookies_infos
```
The indent at the top remains. Please use this rule:
`debrid-link.fr#%#//scriptlet('set-cookie', 'cookies_infos', '1')`

Btw you can commit in the current branch and the current PR.

SKEIDs replied:
Thank you for tell me.

Sergey-Lyapin replied:
@SKEIDs 
I moved the rule to the rest of JS-rules, please take a look.

SKEIDs committed:
Fix

debrid-link.fr###cookies_infos
↓
debrid-link.fr#%#//scriptlet('set-cookie', 'cookies_infos', '1')

Sergey-Lyapin committed:
Merge branch 'master' into patch-7
